workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/SpatialOSBehaviourManager.cs

```diff
@@ -0,0 +1,60 @@
+using System.Collections.Generic;
+using Improbable.Worker;
+using UnityEngine;
+
+namespace Improbable.Gdk.Core
+{
+    public class ReaderWriterImpl
+    {
+        // This is placeholder code.
+    }
+
+    /// <summary>
+    ///     Invokes dispatcher callbacks on GameObjects that represent entities.
+    /// </summary>
+    public class SpatialOSBehaviourManager
+    {
+        public GameObject GameObject;
+        private Dictionary<uint, List<MonoBehaviour>> componentIdToReaderSpatialOSBehaviours;
+        private Dictionary<uint, List<MonoBehaviour>> componentIdToWriterSpatialOSBehaviours;
+        private Dictionary<MonoBehaviour, int> numUnsatisfiedComponents;
+        private List<MonoBehaviour> spatialOsBehavioursToEnable;
+        private List<MonoBehaviour> spatialOsBehavioursToDisable;
+        private Dictionary<uint, Dictionary<MonoBehaviour, ReaderWriterImpl>> readersWriters;
```

Comment:
Do we want to initialise these here or are we planning on doing that in the constructor?

Reply:
The caches will be populated in the constructor. Overall, this file is more a placeholder file (hence the lack of any actual implementation) and we will make a proper pass over it later.